packages/api-explorer/e2e/e2e.spec.ts

```diff
@@ -36,6 +36,7 @@ const v40 = `${BASE_URL}/4.0`
 
 describe('API Explorer', () => {
   beforeEach(async () => {
+    localStorage.clear()
```

Comment:
This shouldn't be here, WIP as to how to clear state for tests without causing failures.